examples/with-angular-thirdpartyemailpassword/README.md

```diff
@@ -0,0 +1,43 @@
+![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
+
+# Angular Example app using SuperTokens React library
+
+# SuperTokens ThirdpartyEmailPassword demo app with Angular
+
+This demo app demonstrates the following use cases:
+
+-   Login / sign up with SuperTokens
+-   Logout
+-   Forgot password flow
+-   Session management
+
+This also showcases how you can use the supertokens-auth-react SDK inside a svelte app.
+
```

Comment:
Add more details about how it actually works.. that on /auth/* route we use react, but in others, we use angluar